frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/local-volume-set-inner.tsx

```diff
@@ -86,7 +87,7 @@ export const LocalVolumeSetInner: React.FC<LocalVolumeSetInnerProps> = (props) =
             className="lso-create-lvs__all-nodes-radio--padding"
             value="allNodes"
             onChange={toggleShowNodesList}
-            description="Selecting all nodes will use the available disks that match the selected filters on all nodes."
+            description={allNodesHelpTxt ?? allNodesSelectorTxt}
```

Comment:
Instead of making this conditional let's pass this as default values in function parameters & we don't need to add conditions.

Reply:
Can you explain - what do you mean here?